<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="btn_Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>3</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>3</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Employee</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearAll">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>400</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Employee">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>770</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#tbl_Employee{background: transparent;}</string>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>Full Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phonenumber</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dataset</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lbl_Capture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>520</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lbl_Capture{
	border: 1px solid black;
	border-radius: 15px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/avatar/default_avatar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_takePhoto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../public/img/takePhoto.png</normaloff>../public/img/takePhoto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearRow">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>400</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temporarySave">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Temporary Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Username">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background: transparent;border:None;border-bottom:1px solid;}</string>
    </property>
    <property name="text">
     <string>Nong Trong Quynh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Email">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background: transparent;border:None;border-bottom:1px solid;}</string>
    </property>
    <property name="text">
     <string>abc@gmail.com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearData">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/backgr/backgr-linear.jfif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Complete">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>640</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_FaceData">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../public/img/camera.png</normaloff>../public/img/camera.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Phonenumber">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>240</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background: transparent;border:None;border-bottom:1px solid;}</string>
    </property>
    <property name="text">
     <string>093866522341</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_Department">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>240</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{background: transparent;border:None;border-bottom:1px solid;}</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marketing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finance</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="dataSet_IMG">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#dataSet_IMG{
	border: 1px solid black;
	border-radius: 15px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>95</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Picture Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HUTECH</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>16</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Employee List</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lbl_Header{background-color:#248f8f }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Close">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>3</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/close.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Minimize">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>3</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/minimize.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lbl_Background</zorder>
   <zorder>btn_ClearAll</zorder>
   <zorder>tbl_Employee</zorder>
   <zorder>lbl_Capture</zorder>
   <zorder>btn_takePhoto</zorder>
   <zorder>btn_ClearRow</zorder>
   <zorder>btn_temporarySave</zorder>
   <zorder>txt_Username</zorder>
   <zorder>txt_Email</zorder>
   <zorder>btn_ClearData</zorder>
   <zorder>line</zorder>
   <zorder>btn_Complete</zorder>
   <zorder>btn_FaceData</zorder>
   <zorder>txt_Phonenumber</zorder>
   <zorder>cbox_Department</zorder>
   <zorder>dataSet_IMG</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_Text</zorder>
   <zorder>line_2</zorder>
   <zorder>label_8</zorder>
   <zorder>lbl_Header</zorder>
   <zorder>btn_Back</zorder>
   <zorder>label</zorder>
   <zorder>lbl_Close</zorder>
   <zorder>lbl_Minimize</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
